api/envoy/extensions/transport_sockets/tls/v3/cert.proto

```diff
@@ -431,6 +431,9 @@ message DownstreamTlsContext {
   // certificate.
   google.protobuf.BoolValue require_client_certificate = 2;
 
+  // If specified, Envoy will request a client certificate.
+  google.protobuf.BoolValue request_client_certificate = 7;
```

Comment:
How does this play with `require_client_certificate`, can you add comment about that?